trunk/SUAVE/Analyses/Aerodynamics/AVL_Inviscid.py

```diff
@@ -165,24 +178,35 @@ def evaluate(self,state,settings,geometry):
         AoA           = conditions.aerodynamics.angle_of_attack
         lift_model    = surrogates.lift_coefficient
         drag_model    = surrogates.drag_coefficient
+        e_model       = surrogates.span_efficiency_factor
         
         # Inviscid lift
-        data_len      = len(AoA)
-        inviscid_lift = np.zeros([data_len,1])
+        data_len        = len(AoA)
+        inviscid_lift   = np.zeros([data_len,1])
+        inviscid_drag   = np.zeros([data_len,1])    
+        span_efficiency = np.zeros([data_len,1]) 
+        
         for ii,_ in enumerate(AoA):
-            inviscid_lift[ii] = lift_model.predict([np.array([AoA[ii][0],mach[ii][0]])]) #sklearn update fix
-            
+            inviscid_lift[ii]   = lift_model.predict([np.array([AoA[ii][0],mach[ii][0]])])  
+            inviscid_drag[ii]   = drag_model.predict([np.array([AoA[ii][0],mach[ii][0]])])
+            span_efficiency[ii] = e_model.predict([np.array([AoA[ii][0],mach[ii][0]])])
+        
+        # Store inviscid lift results     
         conditions.aerodynamics.lift_breakdown.inviscid_wings_lift       = Data()    
         conditions.aerodynamics.lift_breakdown.inviscid_wings_lift       = inviscid_lift
-
         state.conditions.aerodynamics.lift_coefficient                   = inviscid_lift
         state.conditions.aerodynamics.lift_breakdown.compressible_wings  = inviscid_lift
         
-        # Inviscid drag, zeros are a placeholder for possible future implementation
-        inviscid_drag                                                    = np.zeros([data_len,1])        
+        # Store inviscid drag results  
+        ar            = geometry.wings['main_wing'].aspect_ratio
```

Comment:
why is there a random ar here?

Reply:
I was pulling this variable out to compute induced drag later down

Reply:
This change is valid in your other pull request.